### PR TITLE
Added progress bar to the metadata on the overview

### PR DESCRIPTION
**Pull request by PurgingPanda**
#796 

Added simple progress bar using the build in <progress> html tag.

I hope this (minor) addition is usefull.

### CODE REVIEW

**Comment by noamokman** [comment]
Can you provide a print screen?

File: src/renderer/pages/TorrentListPage.js [hunk 1]
```diff
@@ -105,6 +106,11 @@ module.exports = class TorrentList extends React.Component {
 
     return (<div key='metadata' className='metadata'>{elements}</div>)
 
+    function renderProgressBar () {
+      var progress = Math.floor(100 * prog.progress)
+      return (<progress value={progress} max='100'>{progress}%</progress>)
```
Comment (dcposch) [inline src/renderer/pages/TorrentListPage.js]: Digging the simplicity

**Comment by dcposch** [comment]
Here's what it looks like now:

screen shot 2016-08-30 at 8 55 26 pm

I think it's a good start, but needs a bit of CSS.

**Comment by dcposch** [comment]
How about this?

screen shot 2016-08-30 at 8 58 40 pm

Smaller progress bar plus a bit of spacing.

```css
progress {
  width: 60px;
  margin-right: 8px;
}
```

**Comment by dcposch** [comment]
LGTM

**Comment by Goldob** [comment]
It doesn't look that good on Windows though:

image

**Pull request merged**